package.json

```
@@ -34,8 +34,8 @@
     "ui components"
   ],
   "engines": {
-    "node": ">=20",
-    "pnpm": ">=9"
+    "node": "18 || 20",
+    "pnpm": ">=8"
```

Pnpm version stays without change
```suggestion
    "pnpm": ">=9"
```